install-worker.sh

```diff
@@ -110,15 +110,20 @@ if [ "$BINARY_BUCKET_REGION" = "us-east-1" ]; then
     S3_DOMAIN="s3"
 fi
 S3_URL_BASE="https://$S3_DOMAIN.amazonaws.com/$BINARY_BUCKET_NAME/$BINARY_BUCKET_PATH"
-
+S3_PATH="s3://$BINARY_BUCKET_NAME/$BINARY_BUCKET_PATH"
 BINARIES=(
     kubelet
     kubectl
     aws-iam-authenticator
 )
 for binary in ${BINARIES[*]} ; do
-    sudo wget $S3_URL_BASE/$binary
-    sudo wget $S3_URL_BASE/$binary.sha256
+    if which aws >/dev/null; then
```

Comment:
Would this be confusing in the case where someone wants to use a private bucket, but does not have the awscli installed?  Maybe print a message that its defaulting to a public URL -- or add an argument to the script to select what it attempts to do, to make it more clear.

Reply:
Good point. Added print statements explaining the cases, PTAL.